value/binary.go

```diff
@@ -1172,6 +1172,32 @@ func init() {
 		},
 
 		{
+			name:      "flip",
+			whichType: atLeastVectorType,
+			fn: [numType]binaryFn{
+				vectorType: func(c Context, u, v Value) Value {
+					countVec := u.(Vector)
+					count, ok := countVec[0].(Int)
```

Comment:
It makes sense to rotate a matrix by a vector of length equal to the rank of the matrix, but we don't need to do that right now. Let's stick to a scalar, which will arrive either as a length 1 vector or a rank 1 matrix. Either way, data will be of length 1.

The error that's already in the code is fine, but the check is insufficient.